Sources/tart/Commands/Push.swift

```diff
@@ -26,13 +26,19 @@ struct Push: AsyncParsableCommand {
                              """))
   var chunkSize: Int = 0
 
+
+  @Option(help: ArgumentHelp("labels to attach to the VM in key=value format",
```

Comment:
See https://github.com/cirruslabs/tart/pull/1052#issuecomment-2781027725:

```suggestion
  @Option(name: [.customLong("label")], help: ArgumentHelp("additional metadata to attach to the OCI image configuration in key=value format",
```